### PR TITLE
Add account-dashboard Web Component

### PR DESCRIPTION
**Pull request by Publicker**
This pull request includes the addition of the `account-dashboard` web component.

Notes:

- The component has been tested locally by incorporating it into the Portal, and it appears to work as expected.
- A follow-up PR will be created on the MoneyMoov Portal (API solution) to display the new Accounts Dashboard.

### CODE REVIEW

**Comment by sipsorcery** [comment]
Latest changes for this PR can be tested at https://business-dev.nofrixion.com/46.

File: packages/web-components/src/AccountDashboard.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import ReactDOM from 'react-dom/client'
+import { AccountDashboard } from '@nofrixion/components'
+import r2wc from 'react-to-webcomponent'
+import React from 'react'
+
+const DashBoard = r2wc(AccountDashboard, React, ReactDOM, {
```
Comment (agranillonf) [inline packages/web-components/src/AccountDashboard.tsx:6]: As far as I know, the "B" in "Dashboard" is never capitalized.

**Review by agranillonf — APPROVED** [review]

**Review by donalnofrixion — APPROVED** [review]

**Review by sipsorcery — APPROVED** [review]

File: packages/web-components/src/AccountDashboard.tsx (hunk 1 above)
Comment (Publicker) [reply packages/web-components/src/AccountDashboard.tsx]: You're right! Thanks

**Comment by sipsorcery** [comment]
Latest changes for this PR can be tested at https://business-dev.nofrixion.com/46.

**Pull request merged**